[EDU-4299] chore: update tickets guide (#849)
aziontech/docs: src/content/docs/en/pages/guides/open-ticket/open-ticket.mdx

After you log in to your Azion account, you'll be redirected to the **Submit a ticket** page. Click **+ New support ticket** to the right of the search field and fill in the field as follows:

| Field | Description |
| --- | --- |
| Requester* | This field is automatically filled in with your Azion login email. |
| Product* | Select the Azion product from the list. You can choose one product per ticket. |
| Subject* | Add a title that describes your problem to identify your ticket. |
| Type* | Select the reason for opening the ticket:<br /><br /><ul><li>**Incident:** to report an incident.</li><li>**Question:** to clarify your question.</li><li>**Request:** to make a specific request; for example, request a new integration or feature.</li></ul> |
| Description* | Detail your needs in this field. Include as much information as possible about your incident, question, or request to assist agents in understanding how best to support you. |
| + Attach a file<br /> *optional* | Add a file to help describe your ticket. It's recommended to use JPEG, PNG, or PDF file extensions. |
| Phone number | Enter the phone number of the individual responsible for discussing the reported problem. |

> | Allow contact by phone | Enable this check box if you wish to be contacted via the provided phone number, should Azion agents deem it necessary for the case. |

Reviewer: ```suggestion
| Allow contact by phone | Enable this check box if you wish to be contacted via the provided phone number, in case Azion agents find it necessary for the case. |
```
